Inventory.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using InventoryManagementSystem.ProductInfo;
+
+namespace InventoryManagementSystem.InventoryInfo
+{
+    public class Inventory
+    {
+        private readonly int _Id;
+        public static int InventoryIdCounter { get; private set; }
+        public string InventoryName { get; set; }
+        private List<Product> ProductList;
+
+        public Inventory() : this("DefaultInventory")
+        {
+
+        }
+
+        public Inventory(string inventoryName)
+        {
+            InventoryName = inventoryName;
+            ProductList = new List<Product>();
+            _Id = InventoryIdCounter++;
+        }
+
+        public override string ToString() => $"Id {_Id} : Name = {InventoryName}";
+
+        public void AddProduct()
+        {
+            Console.Write("Product name : ");
+            var prodcutName = Console.ReadLine();
+            Console.Write("Product price : ");
+            var prodcutPrice = Console.ReadLine();
+            Console.Write("Product quantity : ");
+            var prodcutQuantity = Console.ReadLine();
+
+            var product = new Product()
+            {
+                ProductName = prodcutName,
+                ProductPrice = Double.Parse(prodcutPrice),
+                ProductQuantity = int.Parse(prodcutQuantity)
+            };
+            ProductList.Add(product);
+        }
+
+        public void ViewProdcuts()
+        {
+            foreach(var product in ProductList)
+            {
+                product.Log();
+            }
+        }
+
+        public bool SearchForProduct(string productName)
+        {
+            foreach (var product in ProductList)
+            {
+                if(product.ProductName == productName)
+                {
+                    product.Log();
```

Comment:
This method `SearchForProduct` responsibility should only be about trying to find a product, it shouldn't be printing the thing, maybe you can return the product to the consumer to do whatever he wants to do with it, this way this method will be more generic and can be used for different cases ;)